pylintrc

```diff
@@ -9,6 +9,7 @@ disable=
  E0611, # No name 'config' in module 'redmine' (no-name-in-module)
  R0912, # Too many branches (14/12) (too-many-branches)
  R1702, # Too many nested blocks (6/5) (too-many-nested-blocks)
+ fixme,
```

Comment:
Why is it needed?

Reply:
the linter complains for TODO comments. See https://pylint.pycqa.org/en/latest/user_guide/messages/warning/fixme.html